### PR TITLE
UI - Individual's full name and preferred name

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/20755

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-149-jjub4vsc.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-149-jjub4vsc.web.app

File: btr-web/btr-main-app/components/individual-person/add-new/index.vue [hunk 1]
```diff
@@ -14,34 +14,57 @@
...
-          <div class="pt-3">
-            <BcrosInputsNameField
-              id="individual-person-preferred-name"
-              v-model="significantIndividual.profile.preferredName"
-              name="preferredName"
-              :label="$t('labels.preferredName')"
-              :variant="preferredNameInvalid ? 'error' : 'bcGov'"
-              data-cy="testPreferredName"
+        </UForm>
+        <UForm
```
Comment (hfekete) [inline btr-web/btr-main-app/components/individual-person/add-new/index.vue:35]: Was there no way around without adding additional form ?
We are trying to keep number of forms down to minimum, and eventually have only one if possible

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-149-jjub4vsc.web.app

**Review by hfekete — APPROVED** [review]
Cool, thanks !

**Pull request merged**